assets/multimodal.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- type: dataset
+  name: Multimodal C4
+  organization: Allen Institute for AI
+  description: An augmentation of C4 with images added and made openly available.
+  created_date: 2023-06-09
+  url: https://arxiv.org/pdf/2304.06939.pdf
+  datasheet: ''
+  modality: text and images
+  size: 43B English tokens with 101.2M documents and 571M images
+  sample: []
+  analysis: Conducted experiments on models trained with Multimodal C4 in comparison
+    to models trained on single image/caption datasets
+  dependencies: [C4]
```

Comment:
not sure, but do they say the source for the images?

Reply:
They said the images were derived from public webpages, and then pre-processed. Is that something that should be listed in the dependencies?